stdlib/jwt/src/main/ballerina/src/jwt/jwt_issuer.bal

```diff
@@ -47,13 +47,14 @@ public type JwtKeyStoreConfig record {|
     string keyPassword;
 |};
 
-# Issue a JWT token based on provided header and payload. JWT will be signed (JWS) if `keyStore` information is provided
-# in the `JwtKeyStoreConfig` and the `alg` field of `JwtHeader` is not `NONE`.
+# Issues a JWT token based on provided header and payload. JWT will be signed (JWS) if `keyStore` information is
+# provided in the `JwtKeyStoreConfig` and the `alg` field of `JwtHeader` is not `NONE`.
+# ```ballerina string|error jwt = jwt:issueJwt(header, payload, config); ```
```

Comment:
@kalaiyarasiganeshalingam Is this get properly rendered? Otherwise shall we update all code-snippet as follows:
```suggestion
# ```ballerina
# string|error jwt = jwt:issueJwt(header, payload, config);
# ```
```

Reply:
we can't change to this format because it will produce a compilation error